python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -63,8 +63,6 @@ if(APPLE)
     list(REMOVE_ITEM TEST_OPS test_detection_map_op)
     list(REMOVE_ITEM TEST_OPS test_dist_se_resnext_dgc)
     list(REMOVE_ITEM TEST_OPS test_dist_se_resnext_sync)
-    list(REMOVE_ITEM TEST_OPS test_dist_se_resnext_async)
```

Comment:
context is missing about `test_dist_se_resnext_async`, so i am not sure whether it is OK to be removed.